docs/software/matlabinteractive.md

```diff
@@ -3,58 +3,225 @@ title: MATLAB Interactive
 ```
 
-To load a specific version, you would use:
+For a specific version of Matlab:
 
 ```
-[me@login01 ~]$ module load matlab/R2021a
-```
+[clusteruser@node16 ~]$ module avail matlab
+
+--- /opt/modulefiles ---
+   matlab/R2021a (L)
 
-while the “matlab” wildcard will load the default version, MATLAB R2021a in this case.
+[clusteruser@node16 ~] module load matlab/R2021a
+```
 
-You should now be able to run MATLAB interactively on login nodes:
+Then simply run Matlab from the command line.
 
 ```
-[me@login01 ~]$ matlab
+[clusteruser@node16 ~] matlab
 MATLAB is selecting SOFTWARE OPENGL rendering.
+
+                            < M A T L A B (R) >
+                  Copyright 1984-2021 The MathWorks, Inc.
+                 R2021a (9.10.0.1602886) 64-bit (glnxa64)
+                             February 17, 2021
+
+ 
+To get started, type doc.
+For product information, visit www.mathworks.com.
+ 
+>> disp 'Hello world!'
+Hello world!
+>> exit
+[clusteruser@node16 ~]
+[clusteruser@node16 ~] exit # to shut down your interactive job
 ```
 
-Or you can run it on compute nodes (using CPU or GPU) interactively using X11 forwarding:
+## Web Application
+
+> Note: You can skip this section if you are planning to install Jupyter [in the next section](#jupyter-notebook). The Jupyter interface includes the Matlab Proxy web application, so you won't have to install it separately.
+
+Matlab has an officially-supported web application, [Matlab Proxy](https://github.com/mathworks/matlab-proxy), that lets you use the familiar Matlab graphical user interface in your browser while running all the computations on the cluster. The web app itself is written in Python. Follow [our documentation](python.md) to set up a miniconda environment, or you can use pip to create a simple virtual environment in the folder `.venv` and install Matlab Proxy like this.
+
+> Note: Folders starting with a `.` are hidden. You can list them using `ls -a`. You don't have to put your virtual environment in a hidden folder -- you can call the folder whatever you want.
 
 ```
-[me@login01 ~]$ srun -p test -N 1 -n 8 --time=04:00:00 --pty --x11 bash
-[me@node01 ~]$ module load matlab/R2021a
-[me@node01 ~]$ matlab
-MATLAB is selecting SOFTWARE OPENGL rendering.
+[clusteruser@node16 ~]$ python -m venv .venv
+[clusteruser@node16 ~]$ source .venv/bin/activate
+(.venv) [clusteruser@node16 ~]$ pip install --upgrade pip
+(.venv) [clusteruser@node16 ~]$ pip install matlab-proxy
 ```
 
+And to re-enter the virtual environment later, just run:
+
+```
+[clusteruser@node16 ~]$ source .venv/bin/activate
+(.venv) [clusteruser@node16 ~]$ # now you are in the virtual environment
 ```
-[me@login01 ~]$ srun -p test -N 1 -n 2 --gres gpu:1 --time=04:00:00 --pty --x11 bash
-[me@gpu01 ~]$ module load matlab/R2021a
-[me@gpu01 ~]$ matlab
+
+> Note: Some developers prefer to manage packages and virtual enrionments using [conda](https://docs.conda.io/projects/conda/en/stable/) instead.
```

Comment:
s/enrionments/environments/

Reply:
Updated in this and future PRs via force-push